### ISSUE TITLE
Links within paragraphs are not identifying correctly as interactive elements

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#2113]
## Description of issue

windows 11
voice access
edge

[Fable video](https://app.makeitfable.com/share/685ec98a816251063689d19d?video=1 ) at 3:29 and 6:06

Describe what is failing
- The link within the paragraph is not identified correctly as an interactive element, making it difficult for voice access users to select it without using grid.

## Potential fix

- Investigate the issue and determine if there is an appropriate fix

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria

- [x] Should this issue be added to the A11y Statement if it cannot be resolved soon?
